packages/assets/package.json

```diff
@@ -15,7 +15,7 @@
     "eslint": "^8.47.0",
     "figma-export-assets": "github:mariohamann/figma-export-assets",
     "rimraf": "^5.0.1",
-    "semantic-release": "^22.0.5",
+    "semantic-release": "^19.0.5",
```

Comment:
What is the reason for the downgrade?

Reply:
the semantic-release version we are using is not compatible with the semantic-release-monorepo current version. At least I think that is the problem

Reply:
Ok, so we should keep an eye on that

Reply:
See here: https://github.com/pmowrer/semantic-release-monorepo/issues/138